charts/fybrik/templates/katalog-connector/katalog-viewer-role.yaml

```diff
@@ -5,11 +5,11 @@ apiVersion: rbac.authorization.k8s.io/v1
 {{- if .Values.clusterScoped }}
```

Comment:
Where is this used? Does a (cluster)rolebinding exist for this role?

Reply:
it was, I changed it.  will remove the file at all.